src/integration-test/groovy/au/org/ala/logger/BasicHealthCheckSpec.groovy

```diff
@@ -326,4 +328,15 @@ class BasicHealthCheckSpec extends Specification {
         assert controller.response.json.lastYear.numberOfEvents == 27 && controller.response.json.lastYear.numberOfEventItems == 105
         assert controller.response.json.all.numberOfEvents == 44 && controller.response.json.all.numberOfEventItems == 180
     }
+
+    /**
+     * Returns year and month of a Date
+     * @param inDate Date passed in
+     * @return String of yyyyMM
+     */
+    private String getYearnMonth(Date inDate) {
```

Comment:
Is the `n` between `Year` and `Month` deliberate? I would prefer to see `getYearAndMonth` so it is proper camelcase, as the `n` breaks camelcase rule (not capitalised) and reads more like "get yearn month" when you un-camelcase it.

Reply:
I forgot the reason for the `n`, happy to change.